util/opentelemetry-util-http/src/opentelemetry/util/http/__init__.py

```diff
@@ -47,24 +48,28 @@ def get_traced_request_attrs(instrumentation):
     return traced_request_attrs
 
 
-def get_excluded_urls(instrumentation):
+def get_excluded_urls(instrumentation: str) -> ExcludeList:
     excluded_urls = environ.get(
-        _root.format(f"{instrumentation}_EXCLUDED_URLS"), []
+        _root.format(f"{instrumentation}_EXCLUDED_URLS"), ""
```

Comment:
If a user sets `OTEL_PYTHON_EXCLUDED_URLS=/something` and then sets `OTEL_PYTHON_DJANGO_EXLUDED_URLS=""` in order to exclude django from the global URLs, would this logic force force Django instrumentation to use global exclude list still?

Reply:
Yes, that will be the case. If we want `OTEL_PYTHON_DJANGO_EXCLUDED_URLS` to be able to disable excluded URLs entirely, we can move to this instead:

```python
def get_excluded_urls(instrumentation: str) -> ExcludeList:
    excluded_urls = environ.get(
        _root.format(f"{instrumentation}_EXCLUDED_URLS"),
        environ.get(_root.format("EXCLUDED_URLS"), "")
    )

    return parse_excluded_urls(excluded_urls)
```

Reply:
Change applied to always respect instrumentation-specific variable if set.